test/link/wasm/shared-memory/build.zig

```diff
@@ -0,0 +1,100 @@
+const std = @import("std");
+
+pub fn build(b: *std.Build) void {
+    const test_step = b.step("test", "Test");
+    b.default_step = test_step;
+
+    add(b, test_step, .Debug);
+
+    // Enable the following build modes once garbage-collection is implemented properly.
```

Comment:
they look already enabled, no?

Reply:
Silly me forgot to remove this line when I enabled them in the last commit.